blocks/extra-large-promo-block/features/extra-large-promo/default.jsx

```diff
@@ -107,35 +110,21 @@ const ExtraLargePromo = ({ customFields }) => {
 						type
 						promo_items {
 							basic {
+								_id
 								type
 								url
-								resized_params {
-									800x600
-									800x533
-									800x450
-									600x450
-									600x400
-									600x338
-									400x300
-									400x267
-									400x225
+								auth {
+									${RESIZER_APP_VERSION}
```

Comment:
I don't know if any of the other queries were updated in this way... but nice catch.

Reply:
Will have a look - we may need to create a ticket to address missed one